src/go/rpk/pkg/cli/cmd/wasm/template/wasm.go

```diff
@@ -23,7 +23,7 @@ const uppercase = (record) => {
   const newRecord = {
     ...record,
     value: record.value.map((char) => {
-      if (char > 97 && char < 122) {
+      if (char > 96 && char < 122) {
```

Comment:
looks like this is not converting `z` which is 122, so I think we'd want char >= 97 && char <= 122

Reply:
fixed.